wgsl/index.bs

```diff
@@ -3018,6 +3018,7 @@ The name is available for use after the end of the declaration,
 until the end of the [SHORTNAME] program.
 
 A module-scope let-declared constant must be of [=constructible=] type.
+The type must always be specified.
```

Comment:
This is good to have explicitly in the spec prose.

This is already enforced by the grammar rules, but that's probably too subtle.
(global_constant_decl uses a variable_ident_decl which has both the identifer and the type specifier.  The function-scope let-decl has a path with just the identifier.)

Reply:
(and I see you already mentioned that the grammar enforces it already.)